uint8 action

uint8 NULL_ACTION = 0
uint8 TAKEOFF = 1
uint8 EXPLORE = 2
uint8 TRACK = 3
uint8 GOTO = 4
uint8 INSPECT = 5
uint8 LAND = 6
uint8 WAIT_EXECUTION = 7
uint8 EXECUTED = 8
uint8 EMERGENCY_STOP = 9

uint8 START = 100
uint8 NEXT_ACTION = 200

float64[] data